src/DependencyInjection/ProophEventStoreExtension.php

```diff
@@ -172,6 +173,15 @@ private function loadEventStore(string $name, array $options, ContainerBuilder $
 
         if (! empty($options['repositories'])) {
             foreach ($options['repositories'] as $repositoryName => $repositoryConfig) {
+                $repositoryClass = $repositoryConfig['repository_class'] ?? $repositoryName;
+
+                if (! class_exists($repositoryClass)) {
+                    throw new RuntimeException(\sprintf(
```

Comment:
Please remove the leading `\` for consistency

Reply:
:ok_hand: